R/steps.R

```diff
@@ -161,7 +161,8 @@ Cicerone <- R6::R6Class(
         id = private$id
       )
 
-      private$run_once <- run_once
+      if (run_once)
+        private$runs <- 0
```

Comment:
Didn't have time to run this yet sorry, I think we need to initialise `runs` and `run_once` in the `private` environment/list, no?

Reply:
The `run_once` doesn't need to be initialized because we initialize `private$runs`